views-handlebars/src/main/java/io/micronaut/views/handlebars/HandlebarsViewsRenderer.java

```diff
@@ -34,17 +33,18 @@
 import jakarta.inject.Singleton;
 
 /**
- * Renders Views with with Handlebars.java.
+ * Renders Views with Handlebars.java.
  *
  * @author Sergio del Amo
  * @see <a href="https://jknack.github.io/handlebars.java/">https://jknack.github.io/handlebars.java/</a>
  * @since 1.0
  * @param <T> The model type
+ * @param <R> The response type
```

Comment:
should be request instead of response ?

```suggestion
 * @param <R> The request type
```